### ISSUE TITLE
inception v3 predefined

### ISSUE THREAD

**Issue by dineshbvadhia** [galeone/dynamic-training-bench#14]
Wrt #13, is a predefined Inception V3 model planned?

**Comment by galeone** [comment]
No, it's not planned just because I don't need it (yet). However, contributions are welcome! If someone wants to add a predefined model / predefined dataset should feel free to contribute!

I'll let this issue open because common models like inception are something I'd like to have in DyTB (because everyone use it, and one of the aims of the project is to avoid rewriting common tasks/model)
